seed/index.js

```diff
@@ -17,18 +17,19 @@ var Nonprofit = app.models.Nonprofit;
 var Job = app.models.Job;
 var counter = 0;
 var challenges = getFilesFor('challenges');
-var offerings = 2 + challenges.length;
 
```

Comment:
The plus 2 here takes into account nonprofits and jobs seed. Without it you actually have seed exit early before all the seeding completes.